feature-libs/cart/quick-order/components/quick-order/form/quick-order-form.component.html

```diff
@@ -25,4 +29,14 @@
       {{ 'quickOrderForm.listLimitReached' | cxTranslate }}
     </span>
   </div>
+
+  <button
+    [disabled]="
+      form.invalid || !form.get('product')?.value?.length || isSearching
+    "
+    class="btn btn-primary add-button"
```

Comment:
can we remove add-button as it's not being used for anything

Reply:
I am using it for cypress e2e tests

Reply:
Since this form only have one primary button, your cypress selector can be `cy.get('cx-quick-order-form .btn-primary')`

You shouldn't create redundant classes for the sake of cypress